Added is_floating_point for Paddle frontend

arshPratap opened the pull request:
Close #17970

arshPratap commented:
@hmahmood24 please take a look thanks !

ivy-leaves commented:
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

arshPratap commented:
add_frontend_checklist

arshPratap commented:
@theRealBird @hmahmood24 I have rebased the current branch with the latest patches in master and have also updated the task list to the best of my knowledge ...please take another look at this. Thanks a lot !

theRealBird commented:
Hey @arshPratap,
The PR looks good to me! Thanks a lot for contributing to Ivy. 
I do have a request. Before we merge this, could you please resolve the merge conflicts? Thanks!

arshPratap commented:
> Hey @arshPratap, The PR looks good to me! Thanks a lot for contributing to Ivy. I do have a request. Before we merge this, could you please resolve the merge conflicts? Thanks!

Done ! @theRealBird thanks for your review. I have rebased the branch with `master`. PTAL .Thanks again !

arshPratap commented:
@theRealBird have rebased it to the latest changes on  `master` branch . Can this be merged now

arshPratap commented:
@theRealBird sorry for the delay ! Have rebased to the latest commits ! Please take another look. Thanks

theRealBird commented on ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py:
@arshPratap, Thanks for making the changes. I was going through the code once more and I think passing only "float" data-types to a function which checks if a value is a float or not is not the correct approach for testing. We are not checking how does the function behave when the value passed is not a float.

arshPratap replied:
Yup yup..completely true..lemme add in a quick fix for this

arshPratap replied:
@theRealBird could you take a look now ..decided to move with the `numeric` data types as compared to the previous `float` data type

arshPratap commented:
@theRealBird have made the changes you requested .. would love to hearback on any thoughts you have regarding this PR.Thanks

ivy-seed commented:
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

arshPratap commented:
@theRealBird So sorry for the delay..I have rebased the PR to the current changes in the master..Please take a look.Thanks again

theRealBird commented:
Hey @arshPratap, I am sorry for the delay at my end too. 
Not sure how this skipped under my radar. Could you please take care of the conflicts one last time (hopefully)?
I will be careful to merge it in time. Thanks!

theRealBird reviewed (changes requested):
Just need to resolve the merge conflicts. Thanks!

arshPratap commented:
@theRealBird done !!

theRealBird reviewed (changes requested):
Could you please verify if the test you have implemented is passing? Currently it appears to be failing on the CI dashboard. 

Thanks!

arshPratap commented:
@theRealBird thanks for the .. review and so sorry for the late reply..lemme work on failing tests and I will update the thread here regarding the same.
Thanks

arshPratap commented:
@theRealBird fixed the tests..seems that there are some datatypes that are not compatible with paddle (unsigned integers being the case) ..i had to remove the `"valid"` and `"numeric"` data types and replace them

arshPratap commented:
@theRealBird ...the tests have been completed ..seems like the is_floating_point tests are not failing anymore

theRealBird reviewed (approved):
LGTM
